crates/bevy_ui/src/geometry.rs

```diff
@@ -250,15 +250,13 @@ impl Neg for Val {
 
 #[derive(Debug, Eq, PartialEq, Clone, Copy, Error)]
 pub enum ValArithmeticError {
-    #[error("the variants of the Vals don't match")]
-    NonIdenticalVariants,
```

Comment:
We used to have some math functions that allowed arithmetic between identical variants but they were removed, leaving this behind.